code/modules/religion/rites.dm

```diff
@@ -129,11 +129,17 @@
 	..()
 	var/altar_turf = get_turf(religious_tool)
 	var/blessing = pick(
+		prob(0.0001);/obj/item/organ/internal/cyberimp/arm/combat,
 		/obj/item/organ/internal/cyberimp/arm/surgery,
+		prob(150);/obj/item/organ/internal/cyberimp/arm/toolset,
 		/obj/item/organ/internal/cyberimp/eyes/hud/diagnostic,
 		/obj/item/organ/internal/cyberimp/eyes/hud/medical,
-		/obj/item/organ/internal/cyberimp/mouth/breathing_tube,
+		prob(80);/obj/item/organ/internal/cyberimp/mouth/breathing_tube,
+		prob(50);/obj/item/organ/internal/cyberimp/chest/reviver,
 		/obj/item/organ/internal/cyberimp/chest/thrusters,
+		/obj/item/organ/internal/cyberimp/brain/anti_drop,
+		prob(10);/obj/item/organ/internal/cyberimp/brain/anti_stun,
+		/obj/item/organ/internal/eyes/robotic/shield,
 		/obj/item/organ/internal/eyes/robotic/glow,
 	)
```

Comment:
use pick_weight, please for the love of god and man use pick_weight

Reply:
holy shit
absolutely not

Reply:
pick_weight isn't in the documentation, I used what I could find

Reply:
it's in code/__HELPERS/_lists.dm
we wrote it ourselves

![image](https://github.com/tgstation/tgstation/assets/58055496/bb46cade-1905-4349-9ddc-fb7714151af6)

Can always ask in discord if you're unsure of stuff like this (or check other places that do similar work) if you're unsure, many of the helpers we use aren't inbuilt.

Reply:
(format is a list of things to pick paired with the weights to pick them out of the sum total)
![image](https://github.com/tgstation/tgstation/assets/58055496/9de4b1e1-f089-40cd-8c4f-4551d398c1e6)